src/app/components/CardCarousel/CardCarousel.module.css

```diff
@@ -123,6 +123,7 @@
   color: #0070f3;
   text-decoration: underline;
   cursor: pointer;
+  display: none;
```

Comment:
curious about the objective of this change?

this will hide any element with `viewAllLink` css class, no?